dat/gfx/ARTWORK_LICENSE

```diff
@@ -1142,9 +1124,8 @@
        * uninhabited-rocky-magenta-03.png
        * uninhabited-rocky-magenta-04.png
 
- * Author: Fernando L. Copercini "Bisc88" <fernandoluis1732@yahoo.com.br>, License: GPLv3+
+ * Author: Fernando L. Copercini "Bisc88" <fernandoluis1732@yahoo.com.br>, License: GPLv3
```

Comment:
Why remove the + from GPLv3+ here? These are distinct licensing terms. Software licensed GPLv3+ can also be used under the terms of version 4 or later of the GPL, if GNU ever releases another version.

Reply:
It looked like a mistake as all other files are under GPLv3 but I found the commit from the author.